keyboards/1upkeyboards/1up60hse/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define VENDOR_ID       0x6F75 // OU
```

Comment:
In my experience @Wilba6582 usually uses the uppercase characters for this, which would be `0x4F55` (which is still unassigned according to [the site drashna uses](https://www.the-sz.com/products/usbid/index.php?v=0x4F55&p=&n=)), but I don't know how set Wilba is on preferring uppercase vs. lowercase.

Reply:
@noroadsleft It's only a suggestion for people trying to come up with a unique number to use two ASCII codes.

Reply:
@Wilba6582 :+1:

Reply:
Also, higher is better.   IIRC, USB IF is assigning Vendor IDs in the 0x3000 range.